modules/data.atmosphere/DESCRIPTION

```diff
@@ -80,6 +80,6 @@ License: BSD_3_clause + file LICENSE
 Copyright: Authors
 LazyLoad: yes
 LazyData: FALSE
-VignetteBuilder: knitr
+VignetteBuilder: knitr, rmarkdown
```

Comment:
To fix the current  `check/modules` CI failure, add rmarkdown to the Suggests section, between current lines 71 and 72 above. (I thought all packages already had rmarkdown in Suggests, but apparently we missed this one).

Reply:
Thanks for the feedback! I’ve added rmarkdown to the Suggests section as required .